src/end.sol

```diff
@@ -293,56 +296,56 @@ contract End is LibNote {
 
         uint art = tab / rate;
         Art[ilk] = add(Art[ilk], art);
-        require(int(lot) >= 0 && int(art) >= 0);
+        require(int(lot) >= 0 && int(art) >= 0, "End/ink-art-zero");
```

Comment:
"End/ink-art-int-overflow" since technically this is checking whether `lot` and `art` fit in an `int256`?

Reply:
Oh yeah you are right, this should be an "overflow" message